api/machine/machine.proto

```diff
@@ -270,6 +271,20 @@ message ListRequest {
   int32 recursion_depth = 3;
 }
 
+// DiskUsageRequest describes a request to list disk usage of directories and regular files
+message DiskUsageRequest {
+  // RecursionDepth indicates how many levels of subdirectories should be
+  // recursed.  The default (0) indicates that no limit should be enforced.
+  int32 recursion_depth = 1;
+  // All write counts for all files, not just directories.
```

Comment:
`Report all files, not only directories?` (I think `counts` is wrong, as we report `sizes`)